### ISSUE TITLE
unusual use of CPU

### ISSUE THREAD

**Issue by abazvpo** [elementary/mail#498]
On the idle mode (when the app is closed), the pantheon-mail is using about 50%-60% of CPU and draining the battery a lot. Consequently, the laptop is getting warm (=> fan noise...etc.). However, when I kill the process, everything "calms down" and the CPU usage comes back to its normal state. 
It may have something to do with the notification system perhaps, that is, if "always watch for new email" is enabled, the problem persists, if it's disabled, the problem disappears.
Any ideas on how to fix it?

**Comment by janxkoci** [comment]
I don't see any high CPU usage on my two laptops, so I don't know where this is comming from. When I run Mail in the background, I notice in taska manager it sometimes checks for mail and the CPU usage increases to something like 40-50 %, but only for a short time and then it gets low again, in the range of 1-2 %. It doesn't stay high all the time. I'm using Gmail and our university email and the checks are quite infrequent, maybe your provider is different.

What is your mail provider?

However, I can share a tip that works for me to solve similar issue (not CPU, but memory). One of my laptops is a low-spec netbook with only 2 GB of RAM and running Mail in the background was expensive for me. But since I get notifications on my phone anyway, I just turned off the "check mail in background" setting. The app starts and syncs pretty fast, so whenever I get notification on the phone, but wanna read the mail on bigger screen, I open the app and in less than 10s I can read the mail there.

So basically, if the background service causes you trouble, you can turn it off and use your phone for notifications, while still using the app for actually reading and writing emails.

**Comment by marbetschar** [comment]
@abazvpo I suspect the behavior is taking place on Hera. As mentioned in the other issue Mail is undergoing a major rework - therefore the original information does probably no longer reflect the current state of Mail.

Closing this. Feel free to open a new issue if there are any problems on Odin with the new code base.
